update_requirements.sh

```diff
@@ -3,11 +3,21 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 cd $DIR
 
+RC_FILE="${HOME}/.$(basename ${SHELL})rc"
```

Comment:
this isn’t always the case. mac defaults to ~/.bash_profile

Reply:
.bash_profile normally calls .bashrc so I thought this is a superset, didn't realize mac doesn't do this :(